From db6f7d01f6826a5a34dd901890f922edf60ef4c9 Mon Sep 17 00:00:00 2001
From: andip71 <andreasp@gmx.de>
Date: Wed, 25 Nov 2015 11:44:38 +0100
Subject: [PATCH] 2

---
 init.rc | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/init.rc b/init.rc
index 4b644bc..9a8ab4e 100755
--- a/init.rc
+++ b/init.rc
@@ -765,3 +765,10 @@ service logcatd /system/bin/logcat -b all -v threadtime -v usec -v printable -D
     user logd
     group log
     writepid /dev/cpuset/system-background/tasks
+
+# Call Boeffla-Kernel init script
+service boeffla-init /sbin/boeffla-init.sh
+    class main
+    user root
+    group root
+    oneshot
-- 
1.9.1

